<ui version="4.0" >
 <class>OmgExperimentSelectorBase</class>
 <widget class="QDialog" name="OmgExperimentSelectorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>openModeller Desktop - Experiment Selector</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/om_icon_small.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Experiments</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="lstExperiments" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Pre-Analysis</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="lstPreAnalysis" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lstExperiments</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>lstPreAnalysis</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OmgExperimentSelectorBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>363</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel" >
     <x>417</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OmgExperimentSelectorBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>286</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel" >
     <x>235</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
